adaptive_navigation/pubspec.yaml

```diff
@@ -4,12 +4,14 @@ version: 0.0.1
 homepage: https://github.com/material-components/material-components-flutter-adaptive
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: ">=2.12.0-0 <3.0.0"
 
 dependencies:
   flutter:
     sdk: flutter
-  adaptive_breakpoints: ^0.0.2
+  adaptive_breakpoints:
+    # TODO: Use pub dependency once NNBD beta is released.
+    path: ../adaptive_breakpoints
```

Comment:
Will this actually work if someone is using the `adaptive_navigation` pub package?